docs/integrations/integrations.md

```diff
@@ -0,0 +1,128 @@
+
+In our case, we want to register an instance of the `oci-registry` integration. To do so, we need to first figure out what configuration parameters are required by the integration. We can do so by running:
+
+```sh
+$ chainloop integration available describe --id oci-registry
+┌──────────────┬─────────┬──────────────────────┬────────────────────────────────────────────────┐
+│ ID           │ VERSION │ MATERIAL REQUIREMENT │ DESCRIPTION                                    │
+├──────────────┼─────────┼──────────────────────┼────────────────────────────────────────────────┤
+│ oci-registry │ 1.1     │                      │ Send attestations to a compatible OCI registry │
+└──────────────┴─────────┴──────────────────────┴────────────────────────────────────────────────┘
+┌──────────────────────────────────────────────────────────────┐
+│ Registration inputs                                          │
+├────────────┬────────┬──────────┬─────────────────────────────┤
+│ FIELD      │ TYPE   │ REQUIRED │ DESCRIPTION                 │
+├────────────┼────────┼──────────┼─────────────────────────────┤
+│ password   │ string │ yes      │ OCI repository password     │
+│ repository │ string │ yes      │ OCI repository uri and path │
+│ username   │ string │ yes      │ OCI repository username     │
+└────────────┴────────┴──────────┴─────────────────────────────┘
+┌─────────────────────────────────────────────────────────────────────────┐
+│ Attachment inputs                                                       │
+├────────┬────────┬──────────┬────────────────────────────────────────────┤
+│ FIELD  │ TYPE   │ REQUIRED │ DESCRIPTION                                │
+├────────┼────────┼──────────┼────────────────────────────────────────────┤
+│ prefix │ string │ no       │ OCI images name prefix (default chainloop) │
+└────────┴────────┴──────────┴────────────────────────────────────────────┘
+```
+
+In the console output we can see a registration section that indicates that 3 parameters are required, let's go ahead and register it using our Google Artifact Registry Credentials by running:
+
+```sh
+$ chainloop integration registered add oci-registry \
+    # i.e us-east1-docker.pkg.dev/my-project/chainloop-cas-devel
+    --opt repository=[region]-docker.pkg.dev/[my-project]/[my-repository] \
+    --opt username=_json_key \
+    --opt "password=$(cat service-account.json)"
+```
+
+> Note: You can find more examples on how to configure this specific integration [here](https://github.com/chainloop-dev/chainloop/tree/main/app/controlplane/plugins/core/oci-registry/v1)
+
+### Attaching an integration
+
+Once the integration is registered, we can attach it to any workflow. In practice this means that attestations and material information received by this workflow will be sent to the registered integration. 
```

Comment:
received by the workflow or generated in the workflow

Reply:
good point, I like the latter.

I´ĺl update in both places

Thanks